content/product/discover/users.md

```diff
@@ -66,12 +66,12 @@ Your partners likely have an incomplete view of the current state or what their
 
 - [How to conduct user research [1]](https://methods.18f.gov/): 18F Methods provide instruction on several different research activities to consider, including [interviews](https://methods.18f.gov/stakeholder-and-user-interviews), [contextual inquiries](https://methods.18f.gov/contextual-inquiry), [heuristic evaluation](https://methods.18f.gov/heuristic-evaluation), and more.
 
-- [How to conduct user research [2]](https://handbook.tts.gsa.gov/research-guidelines/): Guidelines on how research is done at TTS, including informing and getting consent, and managing Personally Identifiable Information (PII).
+- [How to conduct user research [2]](https://guides.18f.gov/ux-guide/research/): Guidelines on how research is done at TTS, including informing and getting consent, and managing Personally Identifiable Information (PII).
```

Comment:
This guide appears to be particular to 18F, rather than TTS, so the language about TTS in this paragraph should probably be changed.

Honestly it feels awkward that we have two different user research guides.

Reply:
updated! took out the reference to tts and changed the description to better reflect the topics in the 18f guide.

yes, it is a bit awkward that we have multiple guides...

Reply:
I would keep the link to TTS but maybe add the UX guide as a separate resource? The UX guide covers a lot more of the "how to do" not just "how to plan." If helpful, I can draft copy.